.github/workflows/gmc-e2e.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+name: gmc e2e test
+
+on:
+  pull_request:
+    branches: [master]
+    types: [opened, reopened, ready_for_review, synchronize] # added `ready_for_review` since draft is skipped
+    paths:
+      - microservices-connector/**
+      - .github/workflows/gmc-e2e.yaml
+  workflow_dispatch:
+
+# If there is a new commit, the previous jobs will be canceled
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  kind:
+    runs-on: k8s
+    steps:
+      - name: Checkout out Repo
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Run e2e test for microservices-connector
+        run: |
+          .github/workflows/scripts/e2e/gmc_test.sh validate_chatqna
+      - name: Clean up e2e test for microservices-connector
+        run: |
+          kubectl delete ns chatqa
```

Comment:
Please use a random suffix in your namespace. Because there will be several tests running in parallel in a same K8s cluster.

Reply:
Does that means we may have multiple gmc-manager instances running at the same time? @irisdingbj will there be any issues in this case?

Reply:
there will be only one gmc-manager which will support multiple CRs